templates/partners/become-a-partner.html

```diff
@@ -0,0 +1,484 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-top">
+      <div class="row">
+        <div class="col-12">
+          <h1>Become a partner</h1>
+          <p class="u-sv3">Partnering with Ubuntu and Canonical consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+        </div>
+      </div>
+      <div class="row u-hide--small">
```

Comment:
Seems odd to hide all of this content on small screens

Reply:
@anthonydillon ok. I will keep the images on mobile.